os/net/socket/bsd_socket_api.c

```diff
@@ -70,6 +70,8 @@
 #include <net/lwip/ip6_addr.h>
 #endif
 
+#define DISABLE_LWIP_LIBC	0
```

Comment:
1. There are two modules, lwip's and new libc's.
    Does `DISABLE_LWIP_LIBC` mean disabling lwip's or disabling libc's?
    It is difficult to know from defintion name.

2. When `DISABLE_LWIP_LIBC` is 0, only libc will be built and be used.
   But when it is 1, there is no way to block libc's module. Both will be built and it will show compilation errors. Isn't it?

Reply:
3. Is it enough to use a local definition, not config? Do you want to not to control it by user?

Reply:
1. DISABLE_LWIP_LIBC means disabling lwip's. It's confuising, I will change the name.
2, 3. DISABLE_LWIP_LIBC is a temporary config to disable lwip's. I found network test cases were all passed with a new libc's, but I think it's better to remove lwip ones later after doing more test.